tests/test_unit/test_placeholder.py

```diff
@@ -0,0 +1,2 @@
+def test_placeholder():
```

Comment:
Opened https://github.com/brainglobe/brainglobe-heatmap/issues/23 to build on this PR.